scripts/intfutil

```diff
@@ -12,44 +12,46 @@ from natsort import natsorted
 
 
 PORT_STATUS_TABLE_PREFIX = "PORT_TABLE:"
+PORT_TRANSCEIVER_TABLE_PREFIX = "TRANSCEIVER_INFO|"
 PORT_LANES_STATUS = "lanes"
 PORT_ALIAS = "alias"
 PORT_OPER_STATUS = "oper_status"
 PORT_ADMIN_STATUS = "admin_status"
 PORT_SPEED = "speed"
 PORT_MTU_STATUS = "mtu"
 PORT_DESCRIPTION = "description"
+PORT_OPTICS_TYPE = "type"
 
 
-def db_connect():
-    db = swsssdk.SonicV2Connector(host='127.0.0.1')
-    if db == None:
+def db_connect_appl():
+    appl_db = swsssdk.SonicV2Connector(host='127.0.0.1')
+    if appl_db is None:
         return None
 
-    db.connect(db.APPL_DB)
+    appl_db.connect(appl_db.APPL_DB)
 
-    return db
+    return appl_db
 
 
-def db_keys_get(db, intf_name):
+def db_keys_get(appl_db, intf_name):
```

Comment:
Now that we've renamed the DB objects because we access more than one DB in this file, This function name should also change in a similar fashion to reflect the fact that it is only meant to act on APPL_DB.

This also applies to similar functions: `db_port_status_get()`, `db_port_optics_get()`

Reply:
Updated file per comments.